routes/articles.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+const express = require("express");
+const router = express.Router();
+const articleDB = require("./../db/articles.js");
+
+router.route("/new").get((req, res) => {
+  res.status(200);
+  res.render("articles/new");
+});
+
+router.route("/:title/edit").get((req, res) => {
+  const getArtByTitle = articleDB.getArt(req.params.title);
+  res.status(200);
+  res.render("articles/edit", getArtByTitle);
+});
+
+router
+  .route("/:title")
+  .get((req, res) => {
+    const getArtByTitle = articleDB.getArt(req.params.title);
+    res.status(200);
+    res.render("articles/article", getArtByTitle);
+  })
+  .put((req, res) => {
+    const artTitle = req.params.title;
+    const artBody = req.body.body;
+    const artAuthor = req.body.author;
+    if (artBody && artAuthor) {
+      const editArt = articleDB.edit(artTitle, artBody, artAuthor);
+      res.status(200);
+      res.render("articles/article", editArt);
+    } else {
+      res.status(500);
```

Comment:
Error should redirect to the `/new` route and save the error in a variable outside this function.